fdbclient/BackupContainerFileSystem.actor.cpp

```diff
@@ -891,16 +891,20 @@ class BackupContainerFileSystemImpl {
 		return Optional<RestorableFileSet>();
 	}
 
+	// Get a set of files that can restore the given "keyRangesFilter" to the "targetVersion".
+	// If "keyRangesFilter" is empty, the restore will recover the whole database. It's generally
```

Comment:
> the restore will recover the whole database

In the context of this function, which merely returns a file set for a restore, I think this should say "the file set will cover all key ranges present in the backup".

Reply:
Done.